src/features/embalm/stepContent/hooks/useBundlr.ts

```diff
@@ -96,13 +95,9 @@ export function useBundlr() {
         toast(uploadSuccess());
         return res.data.id;
       } catch (_error) {
-        const error = _error as Error;
-        console.error(error);
-        toast(uploadFailure(error.message));
-      } finally {
         setIsUploading(false);
```

Comment:
Maybe I'm missing another piece of logic here, but it seems like isUploading only gets cleared on an error? But actually, it looks like we never use `isUploading`. So maybe that local state can be removed?
But then I did a global search for `setIsUploading` and there is another one of those in the bundlr store. `src/store/bundlr/actions.ts` line 66. Maybe we can remove that also.

Reply:
I tried to keep the logic as same-y as possible, esp. since I wasn't sure what `isUploading` was meant to be used for (possibly in the future). 

Since it's now apparent that it's more a stray forgotten thing, I'll take it out, with its references. Thanks for looking that up!